app/views/users/registrations/edit.html.erb

```diff
@@ -1,32 +1,52 @@
+<%= stylesheet_link_tag "application", :media => "all" %>
```

Comment:
Last thing. This isnt necessary. Like I said, rails generates things to just work. Take a look at this page: https://github.com/lamtu1/Pokemon_Project/blob/ef3d5e0f807638b934667cbbc1245f7fe117dfa5/app/views/layouts/application.html.erb#L9-L10 We already have both the js and the css linked.